awx/api/views/analytics.py

```diff
@@ -50,7 +51,23 @@ class AnalyticsRootView(APIView):
 
     def get(self, request, format=None):
         data = OrderedDict()
-        data['reports'] = reverse('analytics_reports_list')
+        data['authorized'] = reverse('api:analytics_authorized')
+        data['reports'] = reverse('api:analytics_reports_list')
+        data['report_options'] = reverse('api:analytics_report_options_list')
+        data['adoption_rate'] = reverse('api:analytics_adoption_rate')
```

Comment:
@john-westcott-iv 
most of these urls is not allowed for GET, that's why I didn't write this code. 
Is it ok to list them if they are POST only?